database/migrations/2024_12_10_014534_create_participant_individualized_service_plan.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('participant_individualized_service_plan', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('participant_id')->constrained();
+            $table->string('participant_name');
+            $table->string('client_number');
+            $table->date('review_date');
+            $table->boolean('parenting_skill_development')->default(false);
+            $table->boolean('effective_co_parenting')->default(false);
+            $table->boolean('employment_and_education')->default(false);
+            $table->boolean('child_support')->default(false);
+            $table->boolean('domestic_violence')->default(false);
+            $table->string('service_identified_by_participant')->nullable();
+            $table->text('goal');
+            $table->text('parenting_skills_objective');
```

Comment:
It looks like Parenting Skills Development, Managing Stress and Anger, Child Support Awareness and Information, Effective Co-Parenting, and Father to Father Mentoring are all required parts of the curriculum.  It looks safe to remove them.

For Custody/Visitation, Education and/or Employment, Housing/Transportation, and Child Support Action Goal, I don't think we need a column for "objective" since those are the names of the objectives, but I do think we need them for "strategies", "person responsible", "timelines", and "measure of success".  For example, we would have custody_visitation_strategies, custody_visitation_person_responsible, custody_visitation_timelines, and custody_visitation_measure_of_success.

We will ask Good Dads about Child Support Action Goal - does the case manager fill this out later (it says Leave Blank)?